files-csharp/CopyAllContentInDirectory/CopyAllContentInDirectory/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.IO.Abstractions;
+
+namespace CopyAllContentInDirectory
+{
+    public static class Program
+    {
+        public static string sourceDir { get; set; } = "";
+        public static string destinationDir { get; set; } = "";
+        public static IFileSystem fileSystem { get; set; } = new FileSystem(); // for testing purposes
```

Comment:
The naming is off. Please fix it here in the source code. You don't need to fix it in the article, since you use the method.

Reply:
Im not 100% sure what the issue is i think it may be properties should start with a capital letter?

im going to push that as the change. 

@CodeMazeBlog

Reply:
Yes indeed, properties should be capitalized, thanks!